icons/hotwire.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" ><title>Hotwire</title><path d="m0 24 15.07108-6.1573-4.85618-.45575 10.56463-9.16849-6.1036-.57277L22.09717 0 3.30033 9.64213l7.55757.8867-9.269 7.48735 4.1402.40893z"/></svg>
```

Comment:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" ><title>Hotwire</title><path d="m16.764 6.917-3.48.81L16.32 0 7.236 11.705l4.334-.854-4.087 7.982 2.364-.532L7.456 24l7.51-8.111-2.77.64 4.568-9.612z"/></svg>
```

The improper scaling should be fixed now
![hotwire](https://github.com/simple-icons/simple-icons/assets/35000807/1cbcc665-9f24-4970-b449-80c77b5a7272)